fang/Cargo.toml

```diff
@@ -15,22 +15,28 @@ rust-version = "1.62"
 doctest = false
 
 [features]
-default = ["blocking", "asynk", "derive-error", "postgres", "mysql" , "sqlite", "migrations_postgres", "migrations_sqlite" , "migrations_mysql"]
-blocking = ["dep:diesel", "dep:diesel-derive-enum", "dep:dotenvy", "diesel?/chrono" , "diesel?/serde_json" , "diesel?/uuid"]
-asynk = ["dep:bb8-postgres", "dep:postgres-types", "dep:tokio", "dep:async-trait", "dep:async-recursion"]
+default = ["blocking", "asynk-sqlx", "derive-error", "blocking-postgres", "blocking-mysql" , "blocking-sqlite", "migrations-postgres", "migrations-sqlite" , "migrations-mysql"]
+asynk-postgres = ["asynk-sqlx" , "sqlx?/postgres"]
+asynk-sqlite = ["asynk-sqlx" , "sqlx?/sqlite"]
+asynk-mysql = ["asynk-sqlx"  , "sqlx?/mysql"]
+asynk-sqlx = ["asynk" , "dep:sqlx"]
+asynk = ["dep:tokio", "dep:async-trait", "dep:async-recursion" ]
 derive-error = ["dep:fang-derive-error"]
-postgres = ["diesel?/postgres" , "diesel?/serde_json", "diesel?/chrono" , "diesel?/uuid" , "diesel?/r2d2"]
-sqlite = ["diesel?/sqlite" , "diesel?/serde_json", "diesel?/chrono" , "diesel?/uuid" , "diesel?/r2d2"]
-mysql = ["diesel?/mysql" , "diesel?/serde_json", "diesel?/chrono" , "diesel?/uuid" , "diesel?/r2d2"]
-migrations_postgres = ["migrations"]
-migrations_sqlite = ["migrations"]
-migrations_mysql = ["migrations"]
+blocking = ["dep:diesel", "dep:diesel-derive-enum", "dep:dotenvy", "diesel?/chrono" , "diesel?/serde_json" , "diesel?/uuid", "diesel?/r2d2"]
+blocking-postgres = [ "blocking", "diesel?/postgres"]
+blocking-sqlite = ["blocking", "diesel?/sqlite" ]
+blocking-mysql = [ "blocking", "diesel?/mysql"]
+migrations-postgres = ["migrations"]
+migrations-sqlite = ["migrations"]
+migrations-mysql = ["migrations"]
 migrations = ["dep:diesel_migrations"]
 
 
 [dev-dependencies]
 fang-derive-error = { version = "0.1.0"}
 diesel_migrations = { version = "2.1" , features = ["postgres", "sqlite" , "mysql"]}
+sqlx = {git = "https://github.com/pxp9/sqlx", branch = "main", features = ["any" , "macros" , "runtime-tokio", "postgres", "sqlite", "mysql"]}
```

Comment:
Yes it is crucial, 

We can not use `Option<T> Decode` because is broken in original `sqlx`

Which is used in `FromRow impl` for Task.

So I have fixed in my fork and I have put a pr in sqlx.

As soon as get merge my pr to sqlx project we will switch into it.

Reply:
https://github.com/ayrat555/fang/blob/support-sqlite-mysql-asynk/fang/src/lib.rs#L197-L207

You should take a look this code.

In the comment is mentioned the issue and it mentioned also that is fixed in my fork.

The issue also has a mention about my pr in sqlx.

Reply:
@pxp9 told me that this is necessary indeed.

He has submitted a PR to `sqlx` fixing the relevant error. We will eventually change back to the crates.io version, but if his PR doesn't get merged we will have to use a workaround he found.